src/components/VAudioTrack/layouts/VGlobalLayout.vue

```diff
@@ -38,6 +38,12 @@ export default defineComponent({
       required: true,
     },
   },
+  setup() {
+    const size = 'medium' as const
+    const layout = 'global' as const
+
+    return { size, layout }
+  },
```

Comment:
Curious why to do this if it's not changing ❓

Reply:
Because there's no way to cast to `const` in the template :slightly_frowning_face: Vue JavaScript expressions in templates appear to be stuck on both an old version of JS syntax (no `??` for example) and also can't have TS in them. If we don't case `size` and `layout` here to `const` and pass them directly in the template as `size="medium"` they get passed as plain `string` types which doesn't match the type of the `size` prop (which is limited to the specific size strings). Same for `layout`.

It's a pain but at least with Vue it doesn't incur any significant runtime cost :crossed_fingers:

Reply:
Ah, I see. Thanks for explanining!